guestbook/templates/mypage.html

```diff
@@ -0,0 +1,41 @@
+{% extends "base.html" %}
+
+{% block title %}My page{% endblock %}
+
+{% block content %}
+  <h1>My page</h1>
+
+  {% with comments_count=comments.count %}
```

Comment:
Good thinking to not execute the same database query (count public comments) multiple times!